test/e2e/proxy_test.go

```diff
@@ -811,6 +928,16 @@ func (w *wsWatcher) WaitForApplication(retryInterval, timeout time.Duration, exp
 	return foundApp, err
 }
 
+func (w *wsWatcher) WaitForApplicationDeletion(retryInterval, timeout time.Duration, expectedAppName string) error {
+	err := kubewait.PollImmediate(retryInterval, timeout, func() (bool, error) {
+		defer w.mu.RUnlock()
+		w.mu.RLock()
+		_, present := w.receivedApps[expectedAppName]
+		return present == false, nil
```

Comment:
`present` is already a boolean
```suggestion
		return !present, nil
```

Reply:
Resolved. Thank you!